shell.js

```diff
@@ -97,6 +97,11 @@ require('./src/which');
 //@include ./src/echo
 require('./src/echo');
 
+
+//@include ./src/mktemp
+var _mktemp = require('./src/mktemp');
+exports.mktemp = common.wrap('mktemp', _mktemp, {idx: 1});
```

Comment:
Also, it doesn't seem to me like globbing would be advantageous here (and it would almost always fail, since few files contain `XXX` in their name).